cfpb/cfgov-refresh
# Housing counselor: Clean up and remove unused JS

Turns out we don't need to be validating the ZIP code in the JS since that happens on the backend, as far as I can tell.

## Removals

- `checkZip` and `getURLQueryVariable` methods and tests.

## Changes

- Renames `hud.js` to `hud-util.js`.
- Renames `main.js` to `common.js`, which gets automatically ignored from unit test code coverage since it's initialization or other code that's better suited for integration tests and won't be unit tested.
- Removes incorrect `zipcode == false` logic in template.

## Testing

1. `gulp clean && gulp build`
2. Visit http://localhost:8000/find-a-housing-counselor/
3. Try with a valid ZIP code, an invalid ZIP code, and nothing at all and confirm results list and error notification appears as expected.
